### ISSUE TITLE
typo in page 44

### ISSUE THREAD

**Issue by shizidushu** [probml/pml-book#66]
book 1 page 44. intepretable should be interpretable

**Comment by jmlipman** [comment]
Another typo on page 30. I guess this should be "y" instead of "x"

![image](https://user-images.githubusercontent.com/3540650/112677780-d1ab1c00-8e72-11eb-9e18-4a5c8335310d.png)

**Comment by jmlipman** [comment]
![image](https://user-images.githubusercontent.com/3540650/113034319-a12cf000-919a-11eb-807a-49fe519fce19.png)

**Comment by jmlipman** [comment]
Found 2 more typos. Also, "TB" was not defined.

![image](https://user-images.githubusercontent.com/3540650/113482090-af825100-94a5-11eb-8698-9e28877fde95.png)

**Comment by jmlipman** [comment]
I've also noticed that "iid" is first used here (and in other places), although it is defined much later, in 4.2.1

![image](https://user-images.githubusercontent.com/3540650/113486807-25de7d80-94bd-11eb-9136-d0fa55f3edd4.png)

**Comment by jmlipman** [comment]
This might be another typo. If the model is unsupervised, I guess there should be no $y$

![image](https://user-images.githubusercontent.com/3540650/113486966-fbd98b00-94bd-11eb-9fad-3ba2ecdf4d60.png)

**Comment by murphyk** [comment]
Thanks, I've fixed all these typos.
@jmlipman  Re eqn 4.7. I have been trying to nudge the ML field to follow the convention used in statistics, which is more logical, but it might be too hard to fight this battle. (I gave up  using p(y) for unsupervsed learning in the intro, after it caused too much confusion.) See my comment below:

"In statistics, it is standard to use $\vy$ to represent variables
whose generative distribution we choose to model, and to use $\vx$
to represent exogoneous inputs which are given but not generated.
Thus supervised learning concerns fitting conditional
models of the form $p(\vy|\vx)$,
and unsupervised learning is the special case where $\vx=\emptyset$,
so we are just fitting the unconditional distribution $p(\vy)$.
In the ML literature,
supervised learning treats $\vy$ as generated and $\vx$ as given,
but in the unsupervised case, it often switches
to using   $\vx$ to represent generated variables."
